lib/geometry/polygon.nit

```diff
@@ -151,7 +139,38 @@ class ConvexPolygon
 		return projection
 	end
 
-	# Is this polygon convex ?
+	# Remove  `p` from the vertices, keeping at least 3 vertices
+	fun delete_vertex(p: Point[Float]) do if points.length > 3 then points.remove(p)
+
+	# Does `self` intersects with `other`
+	fun intersects(other: APolygon): Bool is abstract
+
+	# Is `p` contained in `self` ?
+	fun contain(p: Point[Float]): Bool is abstract
+
+	# Add a vertex to `self`
+	fun add_vertex(p: Point[Float]): Bool do
```

Comment:
You should document the return value (and ideally add nitunits) because it is far from evident